tile_generator/package_flags.py

```diff
@@ -263,3 +266,58 @@ def _apply(self, config_obj, package, release):
             'buildpack_order': '(( .properties.{}_buildpack_order.value ))'.format(packagename),
         })
         package['properties'] = properties
+
+class Helm(FlagBase):
+    @classmethod
+    def _apply(self, config_obj, package, release):
+        # Read the helm chart and bundle all required docker images
+        chart_info = helm.get_chart_info(package['path'])
+        for image in chart_info['required_images']:
+            image_name = image.split('/')[-1]
```

Comment:
`os.basepath(x)` should be used https://docs.python.org/2/library/os.path.html#os.path.basename

Reply:
I would agree with you if this was a pathname, but it's not